installation/on_linux_using_snap.md

```diff
@@ -0,0 +1,12 @@
+# On Linux using Snap
+
+To easily install Crystal on a Linux distribution you can use [Snap](https://snapcraft.io/docs/installing-snapd).
+
+```
+sudo snap install crystal
```

Comment:
The instructions on https://snapcraft.io/install/crystal/ubuntu and https://crystal-lang.org/2019/06/19/snapcraft-summit-montreal.html use the `--classic` flag.